### ISSUE TITLE
String attributes not being applied to script element

### ISSUE THREAD

**Issue by dunncl15** [hupe1980/react-script-hook#6]
I'm noticing that string attributes (e.g. 'data-testid') are not added to the script element properly. In the example below I'm loading the google api script, but when I inspect the script element upon load the `data-testid` property is missing.

```
const [loading, error] = useScript({
    src: 'https://apis.google.com/js/api.js',
    type: 'text/javascript',
    'data-testid': 'gapi',
  });
```

I'd like purpose updating the code to use `setAttribute(key, attributes[key]` rather than `scriptEl[key] = attributes[key]`. I'm happy to open a PR for this if you're onboard with the change.

**Comment by hupe1980** [comment]
Do you mean this case?
https://github.com/hupe1980/react-script-hook/blob/1cdca7a9b13925872e2db3979b4f5938dcef656f/src/__tests__/use-script.test.tsx#L32-L55

**Comment by dunncl15** [comment]
I do. I did run your tests locally and they seem to pass, however when implementing this hook elsewhere the `'data-test'` key is not added as an attribute. Even just creating a dummy script tag and trying to add a `'data-test'` key via the console doesn't seem to work. Using `setAttribute` instead of direct assignment did seem to work from what I've seen.

```
const scriptEl = document.createElement('script');
script.src = 'testscript.com';
script['data-test'] = 'test-id';
// scriptEl ==> <script src="testscript.com"></script>
```

**Comment by hupe1980** [comment]
Yes, then let's change that. Setting the attributes via the API is also a bit cleaner. Do you want to create a PR?

**Comment by dunncl15** [comment]
Yeah sure I'll open one shortly.

**Comment by dunncl15** [comment]
@hupe1980 I have a branch with these updates. Can you double check the access rights for the repo? I can't push currently.

**Comment by hupe1980** [comment]
You have to fork my repo, push your change and then create a pr

**Comment by hupe1980** [comment]
Merged #7 into master.

**Comment by lyallh** [comment]
Hi. Unfortunately PR #7 breaks the onload callback.

It seems setAttribute does not handle attributes like onload properly and the event handler doesn't fire.

I'll PR my hotfix to consider that sets only non-standard attributes with setAttribute.
